src/ducks/settings/TagsListSettings.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { makeStyles } from '@material-ui/core/styles'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+import TagsList from 'ducks/settings/TagsList'
+
+import SearchInput from 'components/SearchInput'
+import SelectInput from 'components/SelectInput'
+import { Unpadded } from 'components/Padded'
+
+const useStyles = makeStyles(() => ({
+  mobileBox: {
+    display: 'flex',
+    flexDirection: 'column',
+    padding: '.5rem',
+    gap: '.5rem'
+  },
+  desktopBox: {
+    display: 'flex',
+    flexDirection: 'row',
+    margin: '2rem 2rem 1rem 2rem',
+    gap: '.625rem'
+  }
+}))
+
+const TagsListSettings = ({ tags }) => {
+  const { isMobile } = useBreakpoints()
+  const styles = useStyles()
+  const { t } = useI18n()
+  const [filter, setFilter] = useState('')
```

Comment:
filter looks like a array of filter. It should be called "filters" and default value can be [], I guess